Client/Frontend/Browser/BrowserViewController/BrowserViewController+WKNavigationDelegate.swift

```diff
@@ -258,6 +258,37 @@ extension BrowserViewController: WKNavigationDelegate {
                 self.tabManager.selectedTab?.alertShownCount = 0
                 self.tabManager.selectedTab?.blockAllAlerts = false
             }
+            
+            // Prevents synthetically activated links such as: CVE-2017-7089
+            if ["data", "blob", "file"].contains(url.scheme) {
```

Comment:
@Brandon-T - We should add `javascript:` to this list as well. The test case can be modified to use javascript scheme to exhibit the same behavior.

Reply:
`javascript:` URLs are already blocked because of the previous flaw with bookmarks.

Reply:
I'm able to reproduce using: https://jumde.github.io/test/data.html - Let me know if I'm missing something